pages/references.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Project References</h1>
+    <p class="mb-3">
+      <span class="font-medium">Database Source:</span> Julian Goodare, Lauren
+      Martin, Joyce Miller and Louise Yeoman,
+      <a
+        class="underline hover:text-gray-500"
+        href="http://www.shca.ed.ac.uk/witches/"
+        target="_blank"
+      >
+        'The Survey of Scottish Witchcraft' </a
+      >(archived January 2003)
+    </p>
+    <br />
+    <h2 class="mb-3">Place-Name References</h2>
+    <p class="mb-3">
+      Alexander, W.M.C. & Third Spalding Club (1952). The place-names of
+      Aberdeenshire, Aberdeen: Printed for the Third Spalding Club.
+    </p>
+    <p class="mb-3">Blaeu, J (1654). Blaeu's Atlas novus, volume V.</p>
+    <p class="mb-3">
+      Johnston, J.B. & Royal Scottish Geographical Society (1940). The
```

Comment:
Need to replace the ampersand with `&amp;`

Reply:
done

Reply:
Thanks Ruby